dGame/dComponents/SkillComponent.cpp

```diff
@@ -248,6 +246,8 @@ SkillExecutionResult SkillComponent::CalculateBehavior(const uint32_t skillId, c
 
 	context->caster = m_Parent->GetObjectID();
 
+	context->skillID = skillId;
```

Comment:
Keep this in though.  Since most of the time skillID was only needed for client casts I ignored setting it here.